main.go

```diff
@@ -6,7 +6,7 @@ import (
 	"path"
 	"strconv"
 
-	"github.com/codegangsta/cli"
+	"github.com/docker/machine/cli"
```

Comment:
I think you can push to master on your fork and point at that instead, right?

`github.com/nathanleclaire/cli`